ouroboros-consensus/src/Ouroboros/Consensus/HardFork/Combinator/AcrossEras.hs

```diff
@@ -240,6 +240,8 @@ deriving via LiftNamedMismatch "MismatchEraInfo" SingleEraInfo LedgerEraInfo xs
   Other instances
 -------------------------------------------------------------------------------}
 
+deriving via LiftNS Header            xs instance CanHardFork xs => Eq   (OneEraHeader xs)
```

Comment:
Copy the thing we do for `HardForkBlock`: https://github.com/input-output-hk/ouroboros-network/blob/060547120a8c79957d6e31742d71da735c41ec15/ouroboros-consensus/src/Ouroboros/Consensus/HardFork/Combinator/Block.hs#L173